Automattic/wp-calypso
# Anchor: Add podcast ID, episode ID, and Spotify URL to onboard store

#### Changes proposed in this Pull Request

* Add relevant selectors, reducers, and actions for this data to the store.
* Pass podcast ID, episode ID, and Spotify URL to the onboard store for use in site creation
* Update Design Selection to use store data rather than query params

#### Testing instructions

* Switch to this PR, navigate to `/setup/?anchor_podcast=[your podcast ID]&anchor_episode=12345&spotify_url=something`
* Open dev tools, switch to Application tab
* Click Continue to navigate to the next step in the flow
* The onboard store should have the data you entered as query params:
<img width="844" alt="Screen Shot 2022-05-02 at 11 14 52 AM" src="https://user-images.githubusercontent.com/2124984/166259623-5d40a0d9-c3d6-4d71-8f0e-ffb28bd514b4.png">

* Select a design to complete the flow
* Check your new site's options via the dev console: WP.com API v1.2, `/sites/yoursite.wordpress.com/`; you should have an `anchor_podcast` field under `options` with the podcast ID
